sdk/textanalytics/ai-text-analytics/swagger/README.md

```diff
@@ -235,6 +235,22 @@ directive:
       $["$ref"] = "#/definitions/DetectLanguageInput";
 ```
 
+### Disable LRO
+
+```yaml
+directive:
+  - from: swagger-document
+    where: $["paths"]
+    transform: >
+      for (var path in $) {
```

Comment:
can't you use `where: $["paths"][*]` to select this instead?

Reply:
Done.